test/hotspot/jtreg/compiler/c2/irTests/RShiftINodeIdealizationTests.java

```diff
@@ -27,7 +27,7 @@
 
```

Comment:
Up to you if you want to update the copyright year or add your company's copyright. Same in the other test.